src/main/java/codeu/model/store/basic/HashtagStore.java

```diff
@@ -117,4 +117,16 @@ public Hashtag getHashtag(UUID id) {
     return null;
   }
 
+  public List<Hashtag> getHashtagsFromContent(String title) {
+    String[] hashtagList = title.split(" ");
+    List<Hashtag> hashtagResults = new ArrayList();
+    for (String hash : hashtagList) {
+      if (hash.startsWith("#")) {
+        String hashtagName = hash.substring(hash.indexOf("#")+1);
```

Comment:
nit: since we used startsWith, that means the hash symbol must be the first character of the string. That means this could just be hash.substring(1);